src/content/en/updates/2017/09/auto-play-policy-changes.md

```diff
@@ -0,0 +1,154 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: Learn best practices for good user experiences with the new auto-play policies in Chrome.
+
+{# wf_updated_on: 2017-09-04 #}
+{# wf_published_on: 2017-09-04 #}
+{# wf_tags: news,media #}
+{# wf_featured_image: /web/updates/images/generic/play-outline.png #}
+{# wf_featured_snippet: TODO #}
+{# wf_blink_components: Blink>Media #}
+
+# Auto-Play Policy Changes {: .page-title }
+
+{% include "web/_shared/contributors/beaufortfrancois.html" %}
+
+Chrome's auto-play policies are about to change in 2018 and I'm here to tell
```

Comment:
autoplay is used over auto-play in Chrome.

Reply:
Done.